docs/references.md

```diff
@@ -38,3 +38,13 @@
 
 - [Ralf Gommers - Python packaging in 2021 - pain points and bright spots (2021)](https://labs.quansight.org/blog/2021/01/python-packaging-brainstorm/)
 
+
+## General packaging core concepts
+
+- [Python Packaging Authority - Python Packaging User Guide (2010)](https://packaging.python.org/)
+- [Naomi Ceder, for Manning Books - The Quick Python Book, 3rd edition, Chapter 18 (2018)](https://www.manning.com/books/the-quick-python-book-third-edition)
+- [SciKit-HEP - Packaging (2020)](https://scikit-hep.org/developer/packaging)
+- [Test & Code - Python Packaging - Brett Cannon (2021)](https://podcasts.apple.com/in/podcast/python-packaging-brett-cannon/id1029487211?i=1000520722142)
+- [Tomas Beuzen and Tiffany Timbers - Python Packages (2022)](https://py-pkgs.org/)
+- [Real Python - How to Publish Your Own Python Package to PyPI (2022)](https://realpython.com/pypi-publish-python-package/)
```

Comment:
Probably better to use author name here as well (Geir Arne Hjelle, says the page), rather than Real Python?

Reply:
I did the same for Test & Code --> Brian Okken